client/src/components/train/ModelTable.tsx

```diff
@@ -122,4 +136,21 @@ const ModelTable: React.FC = () => {
   );
 };
 
+type Status = {
+  status: TrainingStatus;
+};
+
+const ModelStatusIndicator: React.FC<Status> = ({status}) => {
+  switch (status) {
+    case TrainingStatus.Waiting:
+      return <Target color="green" />;
+    case TrainingStatus.Training:
+      return <Target color="#ccc" />;
+    case TrainingStatus.Finished:
+      return <Check color="#333" />;
+    case TrainingStatus.Error:
+      return <AlertTriangle color="orange" />;
```

Comment:
Could be worth considering making this red and rethink the use of red for the delete button.

Reply:
I feel like deletion is normally red and warning/caution is normally yellow orange. Why is orange preferrable for deletion here?